test/fixtures/valid-sources/sources.json

```diff
@@ -244,5 +244,56 @@
       "default": false
     }],
     "createdAt": "1532-12-25T18:45:32.389979"
+  },
+  {
+    "versions": ">=7.14",
+    "production": false,
+    "attribution": [
+      {
+        "label": {
+          "en": "The Silmarillion"
+        }
+      }
+    ],
+    "type": "http",
+    "data": "http://example.com",
+    "name": "shireWithoutIds",
+    "legacyIds": [ "Shire", "Shire regions", "Shire Regions" ],
+    "humanReadableName": {
+      "en": "Shire regions",
+      "de": "Auenland",
+      "zh": "夏爾"
+    },
+    "fieldMapping": [
+      {
+        "type": "id",
+        "name": "wikidata",
+        "desc": "Wikidata identifier",
+        "skipCopy": true
```

Comment:
This is the new property that specifies that this identifier is made of common values that could produce false positives.